### PR TITLE
Create load dbt projects docs

### PR DESCRIPTION
**Pull request by alexanderahn**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208011943581298

### CODE REVIEW

File: docs/SQL/extensibility/dependencies.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+You can copy and paste the information into the Create Dependency fields.
+
+### Add a GitHub dependency
+
+To add a GitHub dependency, you must select **GitHub** and provide the Git Repository, Revision, and Sub Directory.
+
+![Add a GitHub Dependency](img/dependencies-github.png)
+
+The Revision must be either a Git tag, commit has, or branch name.
```
Comment (alexanderahn) [inline docs/SQL/extensibility/dependencies.md:60]: @bandikishores on the GitHub Create Dependency menu, do you know what the checkbox "Warn unpinned" does if the user decides to use it?

Comment (bandikishores) [reply docs/SQL/extensibility/dependencies.md:60]: Hey @alexanderahn , this means that users want to point to their GitHub repo without specifying any version/branch/commit. This means we do a copy of the code from GitHub with whatever was the default branch + the latest code. So every time the user runs the project, it is possible we pull a different code (as the remote GitHub we're referring to might have changed and we said we always want the latest) resulting in unexpected behavior for the user on each run. So it's recommended not to enable it.

File: docs/SQL/extensibility/dependencies.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+:::
+
+### Add a DBT dependency
+
+To add a DBT dependency, you must select **DBT Hub** and provide the dbt package and version number.
+
+![Add a DBT Dependency](img/dependencies-dbt.png)
+
+You can find the dbt package and version number from the [dbt Package hub](https://hub.getdbt.com/dbt-labs/dbt_utils/latest/).
```
Comment (bandikishores) [inline docs/SQL/extensibility/dependencies.md:48]: This link is wrong. It should be `https://hub.getdbt.com/dbt-labs/` and then we can say for example to use latest dbt_utils package you can use `https://hub.getdbt.com/dbt-labs/dbt_utils/latest/`

File: docs/SQL/extensibility/dependencies.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+## Use a dependency
+
+Once you've added a dependency, you can use the following entities from them:
+
+- Models
+- Seeds
+- Functions
+- Gems
+- Data Tests
```
Comment (bandikishores) [inline docs/SQL/extensibility/dependencies.md:82]: Even `Sources` from the Dependent project can be used

File: docs/SQL/extensibility/dependencies.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+## Installation on a cluster
+
+Whenever you connect the Models to a cluster, dependencies are automatically installed on your cluster. If the cluster doesn't have the dependency installed yet, Prophecy installs it and restarts the cluster automatically.
+
+## Storage
+
+All of your dependencies are stored at the Project-level. When adding a dependency to a single Model, by default it becomes available to all the other Models within the same Project.
+
+Dependencies are natively saved within your build-system files. E.g. if you're using Scala, they're saved in the **pom.xml** file, if you're using Python, they're saved in the **setup.py** file.
```
Comment (bandikishores) [inline docs/SQL/extensibility/dependencies.md:107]: This is not true for SQL.
For SQL we can say, it is stored in `packages.yml` file. `Dependencies are saved within your packages.yml file`

File: docs/SQL/extensibility/dependencies.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+:::caution Deleting a dependency
+
+Deleting a dependency deletes it within the whole Project, and all inheriting Models. If you're not confident about
+the functionality of the other Models, it's usually better to disable a dependency, instead of deleting it.
+
+:::
+
+## Installation on a cluster
+
+Whenever you connect the Models to a cluster, dependencies are automatically installed on your cluster. If the cluster doesn't have the dependency installed yet, Prophecy installs it and restarts the cluster automatically.
```
Comment (bandikishores) [inline docs/SQL/extensibility/dependencies.md:101]: Cluster restart won't happen for SQL as there are no cluster level packages that get installed. This entire section can be removed. We can just say, that Prophecy will take care of pulling the dependencies automatically when the model is run.

File: docs/SQL/extensibility/dependencies.md (hunk 2 above)
Comment (alexanderahn) [reply docs/SQL/extensibility/dependencies.md:48]: Done.

File: docs/SQL/extensibility/dependencies.md (hunk 1 above)
Comment (alexanderahn) [reply docs/SQL/extensibility/dependencies.md:60]: Ack.

**Pull request merged**